pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,826 @@
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "source": [
+    "CLUSTER_NAME=\"my-cluster\"\n",
+    "CLUSTER_KEYCLOAK_SECRET=\"uuidgen\"\n",
+    "CLUSTER_DESCRIPTION=\"Description of cluster\"\n",
+    "CLUSTER_REGION=\"moc\""
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Define cluster in cluster-scope\n",
+    "\n",
+    "To create base for cluster, start with creating files in cluster-scope which define groups and namespaces used in cluster.\n",
+    "Create file `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/groups/cluster-admins.yaml` which will contain user names of users which will be considered admins of clusters:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "source": [
+    "!mkdir -p cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/groups\n",
+    "input_text=\"\"\"\\\n",
+    "apiVersion: user.openshift.io/v1\n",
+    "kind: Group\n",
+    "metadata:\n",
+    "    name: cluster-admins\n",
+    "users:\n",
+    "    - user_admin_1\n",
+    "    - user_admin_2\n",
+    "\"\"\"\n",
+    "%store input_text  >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/groups/cluster-admins.yaml "
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'input_text' (str) to file 'cluster-scope/overlays/prod/moc/my-cluster/groups/cluster-admins.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In order to have resources recognizable by ArgoCD we have to list them in `kustomization.yaml` files which link resource files together.\n",
+    "Create `kustomization.yaml` file in `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME`:"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "source": [
+    "input_text=\"\"\"\\\n",
+    "---\n",
+    "apiVersion: kustomize.config.k8s.io/v1beta1\n",
+    "kind: Kustomization\n",
+    "\n",
+    "resources:\n",
+    "  - ../common\n",
+    "  - ../../../../base/user.openshift.io/groups/cluster-admins\n",
+    "  - groups/cluster-admins.yaml\n",
+    "\"\"\"\n",
+    "%store input_text  >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/kustomize.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'input_text' (str) to file 'cluster-scope/overlays/prod/moc/my-cluster/kustomize.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Define cluster in ACM\n",
+    "\n",
+    "In order to use ACM with ArgoCD we have to create manifests which defines if cluster will use ArgoCD. This file is going to be located in `acm/overlays/$CLUSTER_REGION/infra/klusterletaddonconfigs`.\n"
```

Comment:
The opposite actually

```suggestion
    ""
    "Since the cluster is being managed by ACM, we leverage ACM's integration capabilities and use it to connect our ArgoCD through it. All we have to do is instruct ACM to enable the cluster to be managed by ArgoCD. This file is going to be located in `acm/overlays/$CLUSTER_REGION/infra/klusterletaddonconfigs`.\n"
```

Reply:
Resolved in latest commit.